incubator/hnc/api/v1alpha2/hnc_config_conversion.go

```diff
@@ -0,0 +1,191 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"strings"
+
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+
+	v1a1 "sigs.k8s.io/multi-tenancy/incubator/hnc/api/v1alpha1"
+)
+
+const (
+	unknownv1a1Prefix = "UnknownV1Alpha1Value-"
+	unknownv1a2Prefix = "UnknownV1Alpha2Value-"
+)
+
```

Comment:
Based on our conversations with Jordan (offline) to use a schema to limit the allowed values in v1a2, this is no longer necessary, right?

* We don't need the unknownv1a1 because the schema in v1a2 will mean that we _have_ to call it "Ignore."
  * Perhaps we can add a condition in v1a2 that there was a bad value in v1a1? That's not needed in this PR though, we can file an issue to follow up.
* We don't need the unknownv1a2 because the schema will prevent that from ever happening

Is that correct?

Reply:
Yes, but I need to test if `unknownv1a1` prefix can still be taken with enum schema validation now.

Reply:
Removed since we use one-way conversion and the enum schema validation is added in #902